pcsx2/CDVD/ThreadedFileReader.cpp

```diff
@@ -0,0 +1,360 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2021  PCSX2 Dev Team
```

Comment:
Maybe `Copyright (C) 2002-2021  PCSX2 Dev Team` for consistency with the others.